packages/js/product-editor/src/components/block-editor/block-editor.tsx

```diff
@@ -54,6 +53,13 @@ export function BlockEditor( {
 }: BlockEditorProps ) {
 	useConfirmUnsavedProductChanges( productType );
 
+	const blockContext = useMemo( () => {
+		return {
+			...context,
+			productType,
```

Comment:
I suppose this is fine, although the context already contains `postType` which in this case would be the same value as `productType`. Could we just continue to use `postType`?
And if not, could we just add `productType` in the Editor component as part of the context instead of creating a specific `useMemo` for it here?

Reply:
Also I know we updated some of the components here to use `productType`, although this may cause confusion in the future. Given a product type is really either `simple`, `variable`, etc... while a post type value is `product` or `product_variation`.

Reply:
Oh. Great catch. I definitely was confused about this (as now evident by me originally advocating for it to be called `productType` in a previous PR).

I will remove this, as `postType` is perfectly good!